### ISSUE TITLE
MDN sends Return-Receipt-To header, usually used for DSN

### ISSUE THREAD

**Issue by funnisquares** [roundcube/roundcubemail#8069]
I've noticed this recently, although Return-Receipt-To is not RFC compliant, it is still widely used, and that is even recognized in [RFC3464](tools.ietf.org/html/rfc3464). Is there a reason for this? This behavior also does not seem to be RFC compliant, since I've seen no mention of this header in [RFC8098](https://datatracker.ietf.org/doc/html/rfc8098).

**Comment by alecpl** [comment]
Hmmm... I guess we could remove it. Thunderbird for example does not use this header. @thomascube ?

**Comment by funnisquares** [comment]
If this is actually an issue, meaning that I am not mistaken in my assertions, this fixes it. In this patch I've made it so DSN sends the non-standard "Return-Receipt-To" header, since it is non-standard I'd understand If it's not desirable for it to be included.
```
Fixes MDN sending both DSN and MDN headers
--- a/program/include/rcmail_sendmail.php
+++ b/program/include/rcmail_sendmail.php
@@ -191,9 +191,11 @@ class rcmail_sendmail
         }

         if ($mdn_enabled) {
-            $headers['Return-Receipt-To']           = $from_string;
             $headers['Disposition-Notification-To'] = $from_string;
         }
+		if ($dsn_enabled){
+            $headers['Return-Receipt-To']           = $from_string;
+		}

         if (!empty($_POST['_priority'])) {
             $priority     = intval($_POST['_priority']);
```

**Comment by alecpl** [comment]
It looks like some old Microsoft extension, but indeed sending it in MDN mode sounds wrong. I'll tag this ticket as a bug, and I think the proper fix will be to remove the header completely.

**Comment by thomascube** [comment]
Can't remember where it came from, probably just looked at other email clients back then. I however agree to remove it.

**Comment by alecpl** [comment]
Fixed via ed17ce58b11.
